SQL/updates/8-9.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `feedback`.`notes` ADD COLUMN `secret` TINYINT(1) NOT NULL DEFAULT '1'  AFTER `server`
```

Comment:
This file should have a description on what it does (See any other SQL update file)

Also not everyone uses feedback (I know for one I dont), so it should just be `ALTER TABLE 'notes'`